experimental/generated/beats/fields.ecs.yml

```diff
@@ -9766,32 +9766,19 @@
       level: extended
       type: keyword
       ignore_above: 1024
-      description: "Identifies the confidence rating assigned by the provider using\
-        \ STIX confidence scales.\nExpected values:\n  * Not Specified, None, Low,\
-        \ Medium, High\n  * 0-10\n  * Admirality Scale (1-6)\n  * DNI Scale (5-95)\n\
+      description: "Identifies\_the\_confidence\_rating\_assigned\_by\_the\_provider\_\
```

Comment:
Not sure about the Underscores here?

Reply:
👀 Thank you, I didn't notice!

It must have been an artifact left over from how I copied/pasted and how my editor saved the whitespaces. After I edited the line to add the space characters using the space bar, the white spaces now render as expected.